src/pytest_springfield/README.md

```diff
@@ -0,0 +1,6 @@
+`dottedname` Module
+===================
+
+This module exists as a dummy module to help test imports of
+entity classes when using dotted-name strings for `EntityField`
+types.
```

Comment:
@six8 is there a better way to do this? I just wanted test-only nested modules that I could import that would not be imported anywhere else.